admin_manual/installation/deployment_recommendations.rst

```diff
@@ -27,8 +27,12 @@ keep individual Nextcloud instances to a manageable size.
 * PHP 5.5+. PHP 5.4 is the minimum supported version; note that it reached 
   end-of-life in September 2015 and is no longer supported by the PHP team. 
   Some Linux vendors, such as Red Hat, still support PHP 5.4.
-  5.6+ is recommended. ``mod_php`` is the recommended Apache module because it 
-  provides the best performance.
+  5.6+ is recommended. PHP 7 is known to offer significan performance
```

Comment:
We recommend PHP 7 (see https://github.com/nextcloud/documentation/pull/246/files) ... 5.6 + is **required** 😉 @MorrisJobke ... we should change that ;)